doc/htmldoc/developer_space/workflows/nest_with_ides.rst

```diff
@@ -410,7 +410,42 @@ Code Completion in Xcode
 We have to create a new target and configure it appropriately:
 
 1. In the menu select: File -> New -> Target....
-1. Make the target a OS X -> Command Line Tool (Next), of type C++ in your project (preselected). We call it ``completion``
-1. Remove all files and folders that are created with the new target.
-1. In the tab "Build Phase" of the new target, under "Compile Sources" add all ``*.h``, ``*.hpp``, ``*.c``, ``*.cc``, ``*.cpp`` files from the list (you can use CMD+a).
-1. Now Xcode generates its index and after that code completion should work.
+2. Make the target a OS X -> Command Line Tool (Next), of type C++ in your project (preselected). We call it ``completion``
+3. Remove all files and folders that are created with the new target.
+4. In the tab "Build Phase" of the new target, under "Compile Sources" add all ``*.h``, ``*.hpp``, ``*.c``, ``*.cc``, ``*.cpp`` files from the list (you can use CMD+a).
+5. Now Xcode generates its index and after that code completion should work.
+
+
+CLion
+-----
+
+This recipe describes how to set up CLion for editing, building and
+running NEST. The description here was tested on Ubuntu 23.
+
+
+Setting up the project
+~~~~~~~~~~~~~~~~~~~~~~
+
+1. :ref:`Clone NEST <git_workflow>` onto your computer or select ``Get from VCS`` from CLion's start window
+2. First setup CMake
+
+.. image:: ../../static/img/clion_cmake_config.png
+
+3. Add the following CMake option: ``-DPython_ROOT_DIR=[path/to/root/of/python/env]``
```

Comment:
Is this the only thing you need to add to the cmake config?

It is also unclear to me how you navigate from the cmake configuration to the run configuration based on the instructions and screenshots